### DISCUSSION TITLE
Edit/delete your own playlists if they haven't been played yet

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15281] [Ideas]
At the moment, you can create copies of other people's playlists which I think is pretty cool, but it would also be nice if you were able to edit and delete your own playlists. I was making a playlist yesterday and I accidently put the wrong map in my playlist and I didn't realize I had done this until after I had created the playlist. At this point, I couldn't really do anything since there's no option for me to edit or delete my unplayed playlists. I think a feature like this would be very useful.

**Comment by peppy** [comment]
Definitely something I agree we need. Maybe even a grace period after creating the room that it can be deleted / reconstructed even if there are plays already.

I think the simplest way to implement this would be to destroy the old room on hitting the edit button. We already have the ability to create a copy of the room, so this could be added with minimal implementation that way.
